documentation/element-api/client-server-rpc.asciidoc

```diff
@@ -84,12 +84,6 @@ If the return value is a JavaScript `Promise`, then a return value will be sent
 
 == Calling Server-side Methods from the Client
 
-You can call a server-side method from the client side using either the `@EventHandler` or `@ClientCallable` annotation.
```

Comment:
I propose to keep this sentence, but only delete `either the `@EventHandler` or `.

BTW, is the `@EventHandler` no more applicable?

Reply:
It is only applicable for lit templates and deprecated, so not promoting it in docs.

Reply:
Deprecated for polymer templates maybe? Alright, let's ignore my comment.